src/Microsoft.VisualBasic/tests/VB/SpecialDirectoriesTests.vb

```diff
@@ -0,0 +1,78 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System.Environment
+Imports Microsoft.VisualBasic.FileIO
+Imports Xunit
+Namespace Microsoft.VisualBasic.Tests.VB
+    Public NotInheritable Class SpecialDirectoriesTests
+        Private Shared ReadOnly Separators() As Char = {
+            IO.Path.DirectorySeparatorChar,
+            IO.Path.AltDirectorySeparatorChar
+            }
+
+        Private Shared Sub CheckSpecialFolder(folder As SpecialFolder, getSpecialDirectory As Func(Of String))
+            Dim LocalFolderPath As String = Environment.GetFolderPath(folder)
+            If LocalFolderPath = "" Then
+                Assert.Throws(Of IO.DirectoryNotFoundException)(Function() getSpecialDirectory.Invoke)
+            Else
+                Assert.Equal(LocalFolderPath.TrimEnd(Separators), getSpecialDirectory().TrimEnd(Separators))
```

Comment:
Consider creating a helper method for `.TrimEnd(Separators)` since there are several calls within this class. And `Separators` can be inlined directly in the one remaining call.